modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductOptions.testcase

```diff
@@ -363,6 +363,55 @@ definition {
 		}
 	}
 
+	@description = "This is a test for COMMERCE-7821. The information under options should be supported by localization"
+	@priority = 5
+	test CanBeLocalizedWithOptionsInformation {
+		property portal.acceptance = "true";
```

Comment:
@jaydawu this test should not be part of the relevant, please change the property and lower the test priority to 3. Thanks!